cmd/exo/cmd/root.go

```diff
@@ -78,46 +90,64 @@ func initConfig() {
 		}
 	}
 
-	envEndpoint := os.Getenv("CLOUDSTACK_ENDPOINT")
-	envKey := os.Getenv("CLOUDSTACK_KEY")
-	envSecret := os.Getenv("CLOUDSTACK_SECRET")
+	envEndpoint := os.Getenv("EXO_ENDPOINT")
+	envKey := os.Getenv("EXO_KEY")
+	envSecret := os.Getenv("EXO_SECRET")
```

Comment:
`EXOSCALE_API_KEY`, `EXOSCALE_API_SECRET`

Reply:
we have many endpoints.

Reply:
yes